core/hw/maple/maple_devs.cpp

```diff
@@ -215,7 +216,12 @@ struct maple_sega_controller: maple_base
 
 			//struct data
 			//3*4
-			w32( 0xfe060f00);
+#if DC_PLATFORM != DC_PLATFORM_ATOMISWAVE
```

Comment:
We prob. want to subclass here instead of using #defines

Reply:
agreed